### PR TITLE
Update handbook for Matt Manela and team structure

### PR DESCRIPTION
**Pull request by mmanela**
Changes
- Add myself to people list
- Update pages for Product Platform group

### CODE REVIEW

File: content/departments/engineering/teams/search/core.md
Comment (eseliger) [inline content/departments/engineering/teams/search/core.md:1]: should we rename this file?

File: content/departments/engineering/teams/source/index.md [hunk 1]
```diff
@@ -18,7 +19,7 @@ For more details, read our [Source strategy & mission](strategy.md)
 
 ## Contact
 
-- #ask-source channel or @source-support in Slack.
+- #discuss-source in Slack.
```
Comment (eseliger) [inline content/departments/engineering/teams/source/index.md]:
```suggestion
- #discuss-source or @source-support in Slack.
```

I think we can keep that, this is the handle people are supposed to tag instead of individuals where possible :)

**Review by eseliger — APPROVED** [review]

File: content/departments/engineering/teams/source/index.md (hunk 1 above)
Comment (mmanela) [reply content/departments/engineering/teams/source/index.md]: Ah good catch thanks

File: content/departments/engineering/teams/search/core.md
Comment (mmanela) [reply content/departments/engineering/teams/search/core.md:1]: Done

File: content/departments/engineering/teams/graph/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Graph Team
```
Comment (ErikaRS) [inline content/departments/engineering/teams/graph/index.md:1]: This can go in a different PR, but this should probably integrate the content from the old code intel page and then that page can be obsoleted

https://handbook.sourcegraph.com/departments/engineering/teams/code-intelligence/

File: content/departments/engineering/teams/search/index.md [hunk 3]
```diff
@@ -1,7 +1,112 @@
-# Search teams
+# Search Platform team
```
Comment (ErikaRS) [inline content/departments/engineering/teams/search/index.md]: Do we want to change the path to search-platform to avoid confusion with the code search team? (This can also happen in another PR)

**Review by ErikaRS — APPROVED** [review]
Thanks for the updates! I commented on a couple of things that can wait until a separate PR if you prefer. Either way, no need to wait for another round of review.

File: content/departments/engineering/teams/search/index.md (hunk 3 above)
Comment (mmanela) [reply content/departments/engineering/teams/search/index.md]: Done

File: content/departments/engineering/teams/graph/index.md (hunk 2 above)
Comment (mmanela) [reply content/departments/engineering/teams/graph/index.md:1]: I started this but there is a TON of content there and I pulled over just what seemed most pertinent. We will need to do a future pass also.
